src/SeleniumLibrary/keywords/waiting.py

```diff
@@ -51,53 +48,44 @@ def wait_for_condition(self, condition, timeout=None, error=None):
 
     @keyword
     def wait_until_page_contains(self, text, timeout=None, error=None):
```

Comment:
Should we document that text is searched as it is in the DOM which may not always be same as it is the UI? This is corner case but happens sometimes.

Reply:
What's the difference in practice? Do you mean that CSS can change the text somehow?

Reply:
I don't know how it happens in practice but there has been several issues complaining l, with proof, that text in the DOM is with small letters and text in UI is in capital letters. And because keyword uses xpath to search the text, the text is searched as it's in the DOM. All the issues that I recall, have always been with text case.

Reply:
CSS can do all kind of text transform (changing case, adding text before/after, ...) and I assume the text to search is the original, not the styled version. That should be tested, though, before we could document it. Such documentation would also belong to the library main level, not to each keyword searching text.

Reply:
OK